ir/src/lib.rs

```diff
@@ -2,6 +2,7 @@ pub use air_script_core::{
     Constant, ConstantType, Expression, Identifier, IndexedTraceAccess, MatrixAccess,
     NamedTraceAccess, TraceSegment, Variable, VariableType, VectorAccess,
 };
+use parser::ast::SourceSection;
```

Comment:
It would be better to access `SourceSection` consistently everywhere, so it should either always be `ast::SourceSection` or `SourceSection`. Since we've been using the former one so far, I would stick with that, and then we don't need this import.